draft-ietf-quic-multipath.md

```diff
@@ -312,6 +312,19 @@ Each endpoint associates a Receiver Packet Number space to each Path Identifier
 that it provides to the peer. Each endpoint associates a Sender Packet Number space
 to each Path Identifier received from the peer.
 
+
+Endpoints use the same Path ID for one specific path in both directions.
+For a client-initiated path, the client decides which Path ID used for the new path,
+it picks one of the server allocated CID with the specified Path ID.
+Then the client send a PATH_CHALLENGE with the chosen CID. If the server receives the PATH_CHALLENGE,
+it picks a Connection ID with the same path ID for sending the PATH_RESPONSE.
+
+The client MUST choose a previously unused Path ID for which both endpoints have already issued at least one connection ID.
+If the server receives a PATH_CHALLENGE before receiving MP_NEW_CONNECTION_ID
+for the specific path, it MAY choose to ignore the PATH_CHALLENGE, or it can
```

Comment:
```suggestion
If the server receives a PATH_CHALLENGE before receiving a MP_NEW_CONNECTION_ID
for the specific path, it MAY choose to ignore the PATH_CHALLENGE, or it can
```